dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -429,3 +476,8 @@ winrt::Microsoft::Windows::AppNotifications::AppNotification Microsoft::Windows:
 
     return notification;
 }
+
+bool Microsoft::Windows::AppNotifications::Helpers::IsWideStringEmptyOrNull(PCWSTR wideString)
+{
+    return wideString == nullptr || wcslen(wideString) > 0;
```

Comment:
This is incorrect. 
It should return (wideString == nullptr) || (wcslen(wideString) == 0)

Reply:
Better yet `return !wideString || (*wideString == L'\0')`

See my other comment